lombok/src/main/java/com/baeldung/lombok/getter/GetterbooleanSameAccessor.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.lombok.getter;
+
+
+import lombok.Getter;
+
+/**
+ * Related Article Sections:
+ * 3.2. Two boolean Fields With the Same Accessor Name
+ *
+ */
+public class GetterbooleanSameAccessor {

```

Comment:
Let's do either GetterBooleanSameAccessor (capital Boolean) or GetterBooleanPrimitiveSameAccessor. The reason is that making it TitleCase all the way through is more readable. (Notice also that I'm suggesting GetterBooleanType for the other in order to alleviate the ambiguity)